v2/samples/dataprotection/v1api20231101/refs/v1api20200601_resourcegroup.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: resources.azure.com/v1api20200601
+kind: ResourceGroup
```

Comment:
Shouldn't need to have a resource group explicitly given in the sample - the testing infrastructure creates one for you, I think. @super-harsh to confirm.

Reply:
I think the standard name is `aso-sample-rg`, which will be redirected to the actual RG if necessary.

Reply:
Yep, We don't need any resource groups to be created in samples. Just use `aso-sample-rg` and that gets replaces with the "test generated resourceGroup"

Reply:
No this is a different RG, needed while creation backup instance. This RG is known as "Snapshot RG" which is provded in Backup Instance payload. This RG is given explicitly by customer so that all the PV(volume) snapshots are stored within it.